tests/puppeteer/README.md

```diff
@@ -43,4 +43,24 @@ docker-compose up --build
 docker-compose exec -e COMMAND="linkchecker" tests bash /tmp/run-tests.sh
 ```
 
+## Smoke tests 
+This campaign includes a non-exhaustive set of tests and it will ensure that the most important functions work.
+
+### Launch all scripts
+If you want to run the links checker test you can run the script **test/smoke/***
```

Comment:
Be careful, you're talking about the linkchecker in here !

Reply:
done thanks